src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -113,49 +113,56 @@ internal sealed class MeterProviderSdk : MeterProvider
                         return;
                     }
 
-                    // Creating list with initial capacity as the maximum
-                    // possible size, to avoid any array resize/copy internally.
-                    // There may be excess space wasted, but it'll eligible for
-                    // GC right after this method.
-                    var metricStreamConfigs = new List<MetricStreamConfiguration>(viewConfigCount);
-                    foreach (var viewConfig in this.viewConfigs)
+                    try
                     {
-                        var metricStreamConfig = viewConfig(instrument);
-                        if (metricStreamConfig != null)
+                        // Creating list with initial capacity as the maximum
+                        // possible size, to avoid any array resize/copy internally.
+                        // There may be excess space wasted, but it'll eligible for
```

Comment:
This doesn't seem to be the correct English.

Reply:
yes. This was not added in this PR. There are more cleanups coming and will fix